### DISCUSSION TITLE
edit data source [Jan 10]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#4869] [Discord Threads]
> 🙋‍♂️ **Soldado**

**Comment by o1lab** [comment]
> 🙋‍♂️ **wkw**  
#### editting data source s not supported currently. It will be probably available in next feature release.  
#  
> 🙋‍♂️ **Soldado**  
#### Thanks for the feedback  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1051321231678378055)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
